crds/lighthouse.jenkins.io_lighthousejobs.yaml

```diff
@@ -2363,8 +2566,6 @@ spec:
                                                   type: object
                                               type: object
                                             type: array
-                                        required:
```

Comment:
Seems like the only thing that is removed from the card, probably we don't have a required marker :shrug: